app/job_creator.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require_relative 'database_manager'
+
+class JobCreator
+  def initialize(job_data)
+    @job_data = job_data
+  end
+
+  def create_job
+    existing_job = Job.find_by(url: @job_data[:url])
+    return if existing_job
+
+    Job.create(
```

Comment:
What if the job is not being saved in the database?

Reply:
Then probably scraper or creator logic is not working

Reply:
going to add test for this

Reply:
add **rescue** in method